pages/profile/index.tsx

```diff
@@ -6,91 +6,23 @@ import { GetServerSidePropsContext } from "next";
 import { Admin } from "../../types/schema";
 import { getAdmin } from "../../firebase/firestore/admin";
 import nookies from "nookies";
-import Button from "@mui/material/Button";
-import {
-  ProfileInfo,
-  FieldInfo,
-} from "../../components/ProfileInfo/ProfileInfo";
 import Icon from "../../assets/Icon";
+import { AdminProfileForm } from "../../components/AdminProfileForm/AdminProfileForm";
 
 type ProfileSettingsPageProps = {
   currentAdmin: Admin;
 };
 
 const ProfileSettingsPage: React.FunctionComponent<ProfileSettingsPageProps> =
   ({ currentAdmin }) => {
-    const aboutData: FieldInfo[] = [
-      {
-        iconName: "nameicon",
-        fieldName: "NAME",
-        fieldValue: "Cindy Zhang",
-      },
-      {
-        iconName: "singleperson",
-        fieldName: "ROLE",
-        fieldValue: "Executive Director",
-      },
-      {
-        iconName: "lastactive",
-        fieldName: "LAST ACTIVE",
-        fieldValue: "October 20, 2021",
-      },
-      {
-        iconName: "datejoined",
-        fieldName: "DATE JOINED",
-        fieldValue: "January 20, 2021",
-      },
-    ];
-
-    const loginInfo: FieldInfo[] = [
-      {
-        iconName: "email",
-        fieldName: "EMAIL",
-        fieldValue: "chloeisnotarealdog@gmail.com",
-      },
-      {
-        iconName: "phone",
-        fieldName: "PHONE #",
-        fieldValue: "(123) 456 - 7890",
-      },
-      {
-        iconName: "password",
-        fieldName: "PASSWORD",
-        fieldValue: "*******",
-      },
-    ];
-
-    const displayinfo = () => {
-      return (
-        <div className={styles.profile}>
-          <div className={styles.namebar}>
-            <h1> Cindo Zhang </h1>
-            <Button variant="contained" className={styles.button}>
-              <Icon type="editpencil" className={styles.editicon}></Icon>
-              Edit
-            </Button>
```

Comment:
another thing i added after break: put a disabled save button beside edit when there is no info edited yet

<img width="815" alt="Screen Shot 2022-02-09 at 10 30 13 PM" src="https://user-images.githubusercontent.com/77181350/153350521-653420a4-7a68-4e90-b44e-0323e9d2df93.png">